src/features/gym/GymPopup.jsx

```diff
@@ -164,6 +164,7 @@ const DropdownOptions = ({
   raid_level,
 }) => {
   const { t } = useTranslation()
+  const { perms } = useMemory((s) => s.auth)
```

Comment:
```suggestion
  const admin = useMemory((s) => s.auth.perms.admin)  
```

I know the way you have it is how it was done in the Nest Popup component, but that isn't the "correct" way and should be fixed there too. For new components though, I would prefer if you could use the way I suggested on Discord. That goes for the rest of the popups too. Thanks!

Reply:
Changes included. I still have to do it for spawnpoints and routes. This ones are going to be more difficult for me since I have to add the menu...